README.md

```diff
@@ -175,15 +175,16 @@ If you would like to change any of the settings prior to building, pass them in
 conan install .. --build=missing --profile macos-appleclang12-amd64 -o NovelRT:buildtests=False -o NovelRT:documentation=False -o NovelRT:buildsamples=False -o NovelRT:buildinterop=False
 ```
 (Please note: the above command shows the only four options we support at the moment - anything else is considered unsupported at this time.)
-
-Then you should be able to configure and build the default setup, like so:
+If you have an intel mac system, then configure the cmake files like so:
 ```
-conan build .. --build-folder .
+cmake ..
 ```
-
-If you would like to build using CMake instead, you can configure and build, like so:
+If you have an apple-silicon (m1 or m2 or etc) system, you must add this flag after the `cmake ..` like so:
```

Comment:
This reads a bit weird - I think it's because of how you've used the command as part of the sentence.

Instead I would probably say something like "you must add an additional flag to the configuration command, like so:"

That way the example is in one place without duplication and the sentence is more cohesive.

Reply:
Fixed